.github/workflows/ci.yml

```
@@ -62,23 +67,23 @@ jobs:
       - name: Run Features
         run: bundle exec cucumber features/stoplight
         env:
-          STOPLIGHT_REDIS_URL: "redis://127.0.0.1:6379/0"
+          STOPLIGHT_REDIS_URL: 'redis://127.0.0.1:6379/0'
           STOPLIGHT_DATA_STORE: ${{ matrix.data_store }}
           CUCUMBER_PUBLISH_ENABLED: true
 
   spec:
     permissions:
       checks: write  # for coverallsapp/github-action to create new checks
       contents: read  # for actions/checkout to fetch code
-    name: Specs on Ruby ${{ matrix.ruby }} with Redis ${{ matrix.redis }} 💚
+    name: Specs on Ruby ${{ matrix.ruby }} with Data Store as ${{ matrix.data-store-image }} 💚
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        data-store-image: [ 'redis:6.2', 'redis:7.4', 'valkey/valkey:8-alpine' ]
```

I wonder if we work with Valkey 7? 

```suggestion
        data-store-image: [ 'redis:6.2', 'redis:7.4', 'valkey/valkey:8-alpine' , 'valkey/valkey:7-alpine' ]
```

---

I've just added it and looks like it works 🎉